src/loaders/api/api.js

```diff
@@ -150,9 +150,9 @@ export function setAPI (agentIdentifier, forceDrain, runSoftNavOverSpa = false)
       var hasCb = typeof cb === 'function'
       handle(SUPPORTABILITY_METRIC_CHANNEL, ['API/createTracer/called'], undefined, FEATURE_NAMES.metrics, instanceEE)
       // Soft navigations won't support Tracer nodes, but this fn should still work the same otherwise (e.g., run the orig cb).
-      if (!runSoftNavOverSpa) handle(spaPrefix + 'tracer', [now(), name, contextStore], ixn, FEATURE_NAMES.spa, instanceEE)
+      if (!runSoftNavOverSpa) handle(spaPrefix + 'tracer', [TimeKeeper.now(), name, contextStore], ixn, FEATURE_NAMES.spa, instanceEE)
```

Comment:
This is a general comment, and not one that needs any action taken ATM

I've noticed that the inst files only need `.now` and the agg files are the only ones to actually use any correction methods.

We should probably follow up in another effort to see if we can split the timekeeper up to have an inst/agg type dichotomy as well to reduce build size impact for unused mechanisms.

inst == `.now`, agg == `correction methods`

Reply:
(or go back to .now() being its own file used in the inst/agg files, and move the timekeeper to the async chunk)

Reply:
(this would also eliminate the need to access the session entity pre-load, since we only need it at rum-call time (in the agg)

Reply:
I'm on board with having un-floored highrestimestamps if there's nothing hurting it. I'll audit the places using timekeeper.now and open a ticket to clean up those.